Tasks/AppCenterDistribute/Strings/resources.resjson/es-es/resources.resjson

```diff
@@ -26,8 +26,8 @@
   "loc.input.help.releaseNotesInput": "Notas de esta versión.",
   "loc.input.label.releaseNotesFile": "Release notes file",
   "loc.input.help.releaseNotesFile": "Seleccione un archivo de texto con codificación UTF-8 que contenga las notas de esta versión.",
-  "loc.input.label.distributionGroupId": "Distribution group ID",
-  "loc.input.help.distributionGroupId": "ID of the distribution group the app will deploy to. Leave it empty to use the default group.",
+  "loc.input.label.destinationId": "Destination ID",
```

Comment:
did you manually change all the resources.resjson? That is not required, only commit the one generated resources file.

Reply:
Ok, will revert those, thanks for the review!